### DISCUSSION TITLE
Permissions issues

### DISCUSSION THREAD

**Discussion by searlekc** [qwbarch/Pixeldex#5] [Q&A]
Using ForgeEssentials for 1.12.2 the only permission that essentials is aware of is pixeldex.command.base which lets players use /pd, but they have no access to subcommands, and adding the other permissions from the list does not properly allow the other commands. Is there a specific permissions plugin advised or tested with?

**Comment by qwbarch** [comment]
I've only tested it on [LuckPerms](https://ore.spongepowered.org/Luck/LuckPerms) which runs on [SpongeForge](https://www.spongepowered.org/downloads/spongeforge/stable/1.12.2). Is it possible for you to try using that?

Having ForgeEssentials not read any of the permissions is a problem though, I'll create an issue and fix it another day when I'm free. Thanks for the report!

**Reply by searlekc** [reply]
Absolutely, I'll try LuckPerms. Thanks for the quick reply!
